code/modules/admin/admin.dm

```diff
@@ -1447,10 +1447,18 @@ var/global/floorIsLava = 0
 		return
 
 	if(check_rights(R_INVESTIGATE))
-		if(!HAS_STATUS(M, STAT_PARA))
+		if(!M.admin_paralyzed)
+			M.visible_message(
+				SPAN_OCCULT("OOC: \The [M] has been paralyzed by a staff member. Please hold all interactions with them until staff have finished with them."),
+				SPAN_OCCULT("OOC: You have been paralyzed by a staff member. Please refer to your currently open ahelp ticket or, if you don't have one, ahelp for assistance.")
```

Comment:
Would suggest using `Adminhelp` rather than `ahelp` just because the verb isn't called ahelp.

Reply:
Updated